plrust/src/lib.rs

```diff
@@ -193,6 +193,25 @@ unsafe fn plrust_call_handler(fcinfo: pg_sys::FunctionCallInfo) -> pg_sys::Datum
     }
 }
 
+#[pg_extern]
+fn list_allowed_dependencies<'a>() -> Result<
+    Option<
+        ::pgrx::iter::TableIterator<
+            'a,
+            (
+                name!(name, String),
+                name!(version, String),
+                name!(features, Vec<String>),
+                name!(default_features, bool),
+            ),
+        >,
+    >,
+    Box<dyn std::error::Error + Send + Sync + 'static>,
+> {
+    let allowed_dependencies = allow_list::get_allowed_dependencies_as_tuple_vector();
+    Ok(Some(TableIterator::new(allowed_dependencies)))
+}
```

Comment:
The only unfortunate thing is I don't think there's an easy way yet to simply yield a TableIterator that yields the equivalent of a Rust struct so I guess this would have to `.map()` the struct back down to a tuple anyway... hmm... annoying... I should fix that. I mean, PL/Rust users aren't exposed to how grody it is, but